contentcuration/contentcuration/frontend/shared/views/form/MultiSelect.vue

```diff
@@ -20,14 +20,14 @@
             {{ getText(item) }}
           </VChip>
         </template>
-        <template #item="{ item, tile }">
-          <Checkbox v-bind="tile.props" class="ma-0">
-            <template #label>
-              <span :class="{ notranslate }" :style="getEllipsisStyle()" dir="auto">
-                {{ getText(item) }}
-              </span>
-            </template>
-          </Checkbox>
+        <template #item="{ item }">
+          <KCheckbox
+            :checked="selections.includes(item)"
+            :label="getText(item)"
+            :value="item"
+            :style="getEllipsisStyle()"
+            :ripple="false"
```

Comment:
There isn't a `ripple` prop for `KCheckbox`. Also, it wouldn't seem this is preserving the `notranslate` class?